aten/src/ATen/native/native_functions.yaml

```diff
@@ -1970,3 +1970,18 @@
 
 - func: meshgrid(TensorList tensors) -> TensorList
   variants: function
+
+# This has a method dispatch to work around circular include problems
+- func: _local_scalar(Tensor self) -> Scalar
+  variants:
+    - function
+    - method
+
+# NB: Does NOT check precondition that numel == 1
+- func: _local_scalar_dense(Tensor self) -> Scalar
+  cpu_half: True
```

Comment:
I would just put a comment here saying this isn't generally supported, don't use it.

Reply:
The assumption being...people copy/paste, they don't read the docs.

Reply:
New comment:

```
# WARNING: Use of cpu_half here is generally not supported; please
# don't use it.
```